components/script/dom/element.rs

```diff
@@ -693,6 +694,30 @@ impl LayoutElementHelpers for LayoutJS<Element> {
         }
     }
 
+    #[allow(unsafe_code)]
+    fn get_lang_for_layout(&self) -> String {
+        unsafe {
+            let mut current_node = Some(self.upcast::<Node>());
```

Comment:
I suggested this should be using `Element` itself, and doesn't seem addressed. If there's any particular reason it can't be done, or you think it's not worth it, please comment about it, otherwise, please do :)